ajax/libs/sinon.js/package.json

```diff
@@ -16,6 +16,15 @@
     "type": "git",
     "url": "https://github.com/sinonjs/sinon"
   },
+  "npmName": "sinon",
+  "npmFileMap": [
+    {
+      "basePath": "pkg",
+      "files": [
+        "**/*"
```

Comment:
@fatso83 Thank you.

@priidikvaikla I think maybe we can use the rule as the followings:
`**/+(sinon|sinon-no-sourcemaps|sinon-ie|sinon-server).*`

If you have any suggestions, please let me know.
Thank you. :)